File: v2/csharp/src/main/csharp/io/swagger/Model/GroupInput.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class GroupInput {
    

    
    public string Id { get; set; }

    

    
    public string Name { get; set; }

    

    
    public bool? IsDefault { get; set; }

    

    
    public bool? IsHidden { get; set; }

    

    
    public string NewWeeklyMailerContent { get; set; }

    

    
    public string NewsletterSubject { get; set; }

    

    
    public bool? Autopublish { get; set; }

    

    
    public int? NumberArticlesPerGroup { get; set; }

    

    
    public int? NumberArticlesPerNewsletter { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class GroupInput {\n");
      
      sb.Append("  Id: ").Append(Id).Append("\n");
      
      sb.Append("  Name: ").Append(Name).Append("\n");
      
      sb.Append("  IsDefault: ").Append(IsDefault).Append("\n");
      
      sb.Append("  IsHidden: ").Append(IsHidden).Append("\n");
      
      sb.Append("  NewWeeklyMailerContent: ").Append(NewWeeklyMailerContent).Append("\n");
      
      sb.Append("  NewsletterSubject: ").Append(NewsletterSubject).Append("\n");
      
      sb.Append("  Autopublish: ").Append(Autopublish).Append("\n");
      
      sb.Append("  NumberArticlesPerGroup: ").Append(NumberArticlesPerGroup).Append("\n");
      
      sb.Append("  NumberArticlesPerNewsletter: ").Append(NumberArticlesPerNewsletter).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/GroupInput.java
package io.swagger.client.model;


import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class GroupInput  {
  
  @SerializedName("_id")
  private String id = null;
  @SerializedName("name")
  private String name = null;
  @SerializedName("is_default")
  private Boolean isDefault = null;
  @SerializedName("is_hidden")
  private Boolean isHidden = null;
  @SerializedName("new_weekly_mailer_content")
  private String newWeeklyMailerContent = null;
  @SerializedName("newsletter_subject")
  private String newsletterSubject = null;
  @SerializedName("autopublish")
  private Boolean autopublish = null;
  @SerializedName("number_articles_per_group")
  private Integer numberArticlesPerGroup = null;
  @SerializedName("number_articles_per_newsletter")
  private Integer numberArticlesPerNewsletter = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getName() {
    return name;
  }
  public void setName(String name) {
    this.name = name;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Boolean getIsDefault() {
    return isDefault;
  }
  public void setIsDefault(Boolean isDefault) {
    this.isDefault = isDefault;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Boolean getIsHidden() {
    return isHidden;
  }
  public void setIsHidden(Boolean isHidden) {
    this.isHidden = isHidden;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getNewWeeklyMailerContent() {
    return newWeeklyMailerContent;
  }
  public void setNewWeeklyMailerContent(String newWeeklyMailerContent) {
    this.newWeeklyMailerContent = newWeeklyMailerContent;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getNewsletterSubject() {
    return newsletterSubject;
  }
  public void setNewsletterSubject(String newsletterSubject) {
    this.newsletterSubject = newsletterSubject;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Boolean getAutopublish() {
    return autopublish;
  }
  public void setAutopublish(Boolean autopublish) {
    this.autopublish = autopublish;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Integer getNumberArticlesPerGroup() {
    return numberArticlesPerGroup;
  }
  public void setNumberArticlesPerGroup(Integer numberArticlesPerGroup) {
    this.numberArticlesPerGroup = numberArticlesPerGroup;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Integer getNumberArticlesPerNewsletter() {
    return numberArticlesPerNewsletter;
  }
  public void setNumberArticlesPerNewsletter(Integer numberArticlesPerNewsletter) {
    this.numberArticlesPerNewsletter = numberArticlesPerNewsletter;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class GroupInput {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  name: ").append(name).append("\n");
    sb.append("  isDefault: ").append(isDefault).append("\n");
    sb.append("  isHidden: ").append(isHidden).append("\n");
    sb.append("  newWeeklyMailerContent: ").append(newWeeklyMailerContent).append("\n");
    sb.append("  newsletterSubject: ").append(newsletterSubject).append("\n");
    sb.append("  autopublish: ").append(autopublish).append("\n");
    sb.append("  numberArticlesPerGroup: ").append(numberArticlesPerGroup).append("\n");
    sb.append("  numberArticlesPerNewsletter: ").append(numberArticlesPerNewsletter).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
